lms/djangoapps/instructor/views/api.py

```diff
@@ -2322,3 +2322,27 @@ def spoc_gradebook(request, course_id):
         'staff_access': True,
         'ordered_grades': sorted(course.grade_cutoffs.items(), key=lambda i: i[1], reverse=True),
     })
+
+
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_query_params('unique_student_identifier')
+@require_level('staff')
+def mark_student_can_skip_entrance_exam(request, course_id):  # pylint: disable=invalid-name
```

Comment:
Not sure which HTTP methods we typically use in the Instructor Dashboard, looking at other endpoints I see POST, but here we are using HTTP GET. Should this be a POST (with a CSRF token)? We are changing state in the database so normally it wouldn't be a GET. I'm not sure what protection ensure_csrf_cookie gives us with respect to cross-site scripting.